modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/dto/v1_0/converter/ObjectEntryDTOConverter.java

```diff
@@ -1075,6 +1075,10 @@ else if (objectField.compareBusinessType(
 				return null;
 			}
 
+			if (serializable instanceof List) {
```

Comment:
I am worried about this. The serializable is coming from the db object (com.liferay.object.model.ObjectEntry) from [here](https://github.com/liferay/liferay-portal/blob/4e8ce2950897609511ce77e59f7a8fd6fc9c6479/modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/dto/v1_0/converter/ObjectEntryDTOConverter.java#L1271). I wonder why there is a DTO object inside a DB object...